nisystemlink/clients/test_plan/models/_test_plan.py

```diff
@@ -86,17 +73,26 @@ class TestPlan(JsonModel):
     estimated_duration_in_seconds: Optional[float] = None
     """The estimated duration of the test plan in seconds."""
 
-    file_ids_from_template: Optional[List[str]] = None
-    """The list of file identifiers inherited from the template."""
+    system_filter: Optional[str] = None
+    """The filter used to select systems for the test plan."""
 
-    execution_actions: Optional[List[ExecutionDefinition]] = None
-    """The execution actions defined for the test plan."""
+    dut_filter: Optional[str] = None
+    """The filter used to select DUTs for the test plan."""
+
+    created_At: Optional[datetime] = None
+    """The date and time when the test plan was created."""
+
+    updated_at: Optional[datetime] = None
+    """The date and time when the test plan was last updated."""
 
-    execution_history: Optional[List[ExecutionEvent]] = None
-    """The execution history of the test plan."""
+    properties: Optional[Dict[str, str]] = None
+    """Additional properties associated with the test plan."""
 
-    dashboard_url: Optional[Dict[str, str]] = None
-    """The URLs for dashboards related to the test plan."""
```

Comment:
You said these are undocumented, but they are documented in the API's swagger and they're used by the web UI. What prompted you to remove them?